extra_foam/gui/windows/pump_probe_w.py

```diff
@@ -138,8 +148,61 @@ def initUI(self):
 
     def initConnections(self):
         """Override."""
-        pass
+        self._ctrl_widget.save_btn.clicked.connect(self.saveToFile)
 
     def closeEvent(self, QCloseEvent):
         self._ctrl_widget.resetAnalysisType()
         super().closeEvent(QCloseEvent)
+
+    def saveToFile(self):
+        # Get the current data
+        if not len(self._queue):
+            return
+        data = self._queue[0]
```

Comment:
This changed in #318, the GUI now receives a `dict` of all the data from the workers. So here we'd now need `self._queue[0]["processed"]`.